src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  align-self: space-between;
+  width: 96px;
+  text-align: left;
+  margin-bottom: 24px;
+  height: 16px;
+
+  &_star {
+    display: flex;
+    background: url("../images/star.svg") no-repeat;
+    background-position: center;
+    vertical-align: bottom;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  .stars .stars__star:last-child {
```

Comment:
this is duplicated and is in product-card.scss. Decide which one you want to use but don't duplicate